training-tracker-service/src/main/java/org/tolking/config/ObjectMapperConfig.java

```diff
@@ -0,0 +1,16 @@
+package org.tolking.config;
```

Comment:
like I previously mentioned, avoid too generic package names. I think they should at least highlight what application it is. I remember you mentioning it was due to full class name for message dto. They should not dictate how to build our application, we should either remove dependency on class name or extract dtos to separate library